### PR TITLE
Feature update 

### PR DESCRIPTION
**Pull request by kentonwho**
Added ability to set default printer and printing multiple copies of the inputted files. Rewrote some of the options to use bash case, to improve readability as more flags get put on.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,8 +15,12 @@ A command line script to print wirelessly in the Oden Intitute at UT Austin from
 
 - `oden_print -h` : help message
 - `oden_print -l` : list available printers on network
+- `oden_print -d` : list your default printer
+- `oden_print -m` : list your ssh machine 
+- `oden_print -# N printer_name file_to_print` : prints N copies of `file_to_print` on printer `printer_name`
 - `oden_print printer_name file_to_print` : prints `file_to_print` on printer `printer_name`
 
+Note that calling without specifying a printer will print on `<printer name>`
```
Comment (ghbrown) [inline readme.md:23]:
```suggestion
Note that calling without specifying a printer will print on `printer name`, which must be set in the function body by the user.
```
Remove angle brackets, and make it clear users will have to specify default printer.

File: readme.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ A command line script to print wirelessly in the Oden Intitute at UT Austin from
 ## How to set up
 
 - place the contents of `oden_print.sh` in your `.bashrc` or `.profile` file
-- in the copied text replace `<username>` with your sysnet username (for example `jsmith`) and `<machine name>` with your machine's name (for example `bernoulli`)
+- in the copied text replace `<username>` with your sysnet username (for example `jsmith`), `<machine name>` with your machine's name (for example `bernoulli`), and `<printer name>` with your default printer's name
```
Comment (ghbrown) [inline readme.md:9]:
```suggestion
- in the copied text replace `<default machine>` with your sysnet username and machine name (for example `jsmith@bernoulli.oden.utexas.edu`), and `<default printer>` with your default printer's name
```

For consistency between script and readme.

File: oden_print.sh [hunk 3]
```diff
@@ -1,19 +1,45 @@
...
-        echo '       -l, list printer names'
-        echo ''
-    elif [[ "$1" == "-l" ]]; then
-        ssh ${machine} lpstat -p
-    else
-        printer=$1
-        shift # discard first argument, $2 is now $1
-        ssh ${machine} lpr -P ${printer} < "$@"
-    fi
+    default=<default printer>
```
Comment (ghbrown) [inline oden_print.sh:5]:
```suggestion
    default_printer=<default printer>
```
Clearer name (just my preference).

File: oden_print.sh [hunk 4]
```diff
@@ -1,19 +1,45 @@
...
+		echo 'flags: -h, help'
+        	echo '       -l, list printer names'
+		echo ' 	     -d, list default printer'
+		echo '	     -m, list machine used for ssh' 
+		echo '       -#, set number of copies'
+        	echo ''
+    	;; 
+    
+    
+    	"-d") 	echo $default ;;
```
Comment (ghbrown) [inline oden_print.sh:25]:
```suggestion
        "-d")   echo $default_printer ;;
```

File: oden_print.sh [hunk 5]
```diff
@@ -1,19 +1,45 @@
...
+	"-m")	echo $machine ;; 
+	"-l") 	ssh ${machine} lpstat -p ;; 
+	*)	
+		if [[ "$1" == "-#" ]]; then
+			copies=$2
+			shift 
+			shift
+    		fi 	
+		if [[ $# == 1 ]]; then 
+			printer=$default
```
Comment (ghbrown) [inline oden_print.sh:35]:
```suggestion
                        printer=$default_printer
```

File: oden_print.sh [hunk 6]
```diff
@@ -1,19 +1,45 @@
-function oden_print {
+unction oden_print {
     # print to an Oden printer wirelessly via SSH to workstation
     # first argument: printer name
     # other arguments: files to be printed
```
Comment (ghbrown) [inline oden_print.sh:4]:
```suggestion
```
Let's remove these. We already have to keep documentation updated in multiple places, and it has help flag built in.

**Review by ghbrown — COMMENTED** [review]
Thanks very much for the improvements, I think the case statement will make managing additions much easier!

As for the changes, it's mostly style and consistency. If you accept the suggested changes and make sure everything still works, etc. I'm all good to merge.

**Comment by kentonwho** [comment]
Looks good!

**Comment by ghbrown** [comment]
Hm, I can't tell if those suggestions were accepted or not. I'd expect to see new commits if so, but I don't see anything.

I'll merge and then fix up if anything is wacky.

**Comment by ghbrown** [comment]
Yeah, looks like none of the suggestions were committed after all. Perhaps you just resolved the conversations instead?

**Comment by kentonwho** [comment]
Yeah I forgot to commit the changes, will keep in mind next time. Did you manage to add in your edits?

**Comment by ghbrown** [comment]
No worries. Yep, I made a few commits after the merge that essentially added those suggestions.

**Pull request merged**